Program.cs

```diff
@@ -1,7 +1,23 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+
+namespace Training {
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter a string: ");
+         string iso = Console.ReadLine ();
+         bool isogram = Isogram (iso);
+         Console.WriteLine ($"{iso} is {(Isogram (iso) ? "" : "not ")}an isogram.");
+      }
+
+      static bool Isogram (string iso) {
+         iso = iso.ToLower ();

```

Comment:
The lower case conversion could be done in the loop itself.

Reply:
Lower case conversion done in loop